### PR TITLE
[ODS-1856] Increase API response page size limit

### PR DESCRIPTION
**Pull request by tnunnerydl**

### CODE REVIEW

File: Application/EdFi.Ods.Api/Services/Controllers/EdFiControllerBase.cs [hunk 1]
```diff
@@ -132,11 +133,13 @@ protected ILog Logger
             [FromUri] UrlQueryParametersRequest urlQueryParametersRequest,
             [FromUri] TGetByExampleRequest request = default(TGetByExampleRequest))
         {
+            var defaultPageSizeLimit = new DefaultPageSizeProvider().GetDefaultPageSizeLimit();
```
Comment (jamessmckay) [inline Application/EdFi.Ods.Api/Services/Controllers/EdFiControllerBase.cs:136]: Should we be injecting the DefaultSizeProvider?

Comment (jamessmckay) [inline Application/EdFi.Ods.Api/Services/Controllers/EdFiControllerBase.cs:136]: Should we be injecting the DefaultSizeProvider instead?

Comment (jamessmckay) [reply Application/EdFi.Ods.Api/Services/Controllers/EdFiControllerBase.cs:136]: Also should we be addressing the composites pages too?

File: Application/EdFi.Ods.Api/Services/Controllers/CompositeResourceController.cs [hunk 2]
```diff
@@ -92,14 +93,15 @@ public virtual IHttpActionResult Get()
                         kvp => rawQueryStringParameters[kvp],
                         StringComparer.InvariantCultureIgnoreCase);
 
-                //respond quickly to DOS style requests (should we catch these earlier?  e.g. attribute filter?)
+                var defaultPageSizeLimit = new DefaultPageSizeProvider().GetDefaultPageSizeLimit();
```
Comment (jamessmckay) [inline Application/EdFi.Ods.Api/Services/Controllers/CompositeResourceController.cs:96]: DIP

File: Utilities/CodeGeneration/EdFi.Ods.CodeGen/Mustache/Controllers.mustache [hunk 3]
```diff
@@ -13,71 +13,73 @@ using EdFi.Ods.Pipelines.Factories;
...
     public partial class {{ControllerClass}} : EdFiControllerBase<
-        {{ResourceReadModel}},
-        {{ResourceWriteModel}},
-        {{EntityInterface}},
-        {{AggregateRoot}},
-        {{PutRequest}},
-        {{PostRequest}},
-        {{DeleteRequest}},
-        {{GetByExampleRequest}}>
+{{ResourceReadModel}},
```
Comment (jamessmckay) [inline Utilities/CodeGeneration/EdFi.Ods.CodeGen/Mustache/Controllers.mustache:33]: The formatting on this mustached is all wonky

File: Application/EdFi.Ods.Api/Services/Controllers/CompositeResourceController.cs (hunk 2 above)
Comment (jamessmckay) [reply Application/EdFi.Ods.Api/Services/Controllers/CompositeResourceController.cs:96]: This should be a dependency on this class.

File: Utilities/CodeGeneration/EdFi.Ods.CodeGen/Mustache/Controllers.mustache [hunk 4]
```diff
@@ -13,19 +13,20 @@ using EdFi.Ods.Pipelines.Factories;
 using EdFi.Ods.Api.Services.CustomActionResults;
 using EdFi.Ods.Api.Services.Queries;
 using EdFi.Ods.Api.Services.Requests;
+using EdFi.Ods.Common.Configuration;
```
Comment (jamessmckay) [inline Utilities/CodeGeneration/EdFi.Ods.CodeGen/Mustache/Controllers.mustache:16]: please revert the white space changes and regenerate the approval tests.

Comment (tnunnerydl) [reply Utilities/CodeGeneration/EdFi.Ods.CodeGen/Mustache/Controllers.mustache:16]: Done.

**Pull request merged**